scripts/mkcloud

```diff
@@ -667,6 +667,20 @@ function testsetup
     return $ret
 }
 
+function testpreupgrade
+{
+    onadmin testpreupgrade
+
+    return 0
```

Comment:
the failures are checked during the ping verification and execution should abort at that point.